api/service.py

```
@@ -593,6 +653,93 @@ async def get_anomaly_history(
 
+# Authentication endpoints
+@app.post("/api/v1/auth/login", response_model=TokenResponse)
+async def login(request: LoginRequest):
+    """Authenticate user and return JWT token."""
+    auth_manager = get_auth_manager()
+    token = auth_manager.authenticate_user(request.username, request.password)
+    return TokenResponse(access_token=token, token_type="bearer")
+
+
+@app.post("/api/v1/auth/users", response_model=UserResponse)
+async def create_user(request: UserCreateRequest, current_user: User = Depends(require_admin)):
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)